styles/main.css

```diff
@@ -24,6 +24,10 @@
   outline: 0;
 }
 
+#btn-openair {
+  display: none;
+}
+
 .button--save {
   /* position: fixed; */
```

Comment:
can you remove useless stuff?

Reply:
I'm using the 'display:none' for displaying the open air button only when the timesheet is in view but removed other unnecessary stuff.